ProcessSlave.php

```diff
@@ -384,18 +391,20 @@ protected function handleRequest(\React\Http\Request $request, HttpResponse $res
             );
             $this->shutdown();
         }
+        return $response;
     }
 
-    protected function prepareEnvironment(\React\Http\Request $request)
+    protected function prepareEnvironment(RequestInterface $request)
     {
         $_SERVER = $this->baseServer;
```

Comment:
All stuff mentioned there is probably true. We use PHP-PM with Magento project and truly there is a lot of code using SERVER vars. We don't use concurrent-requests there. I think that we should check it again after ServerRequestInterface is ready.